internal/pkg/template/input/file.go

```diff
@@ -17,11 +17,13 @@ func Path() string {
 }
 
 type file struct {
+	Components  []string    `json:"components"`
 	StepsGroups StepsGroups `json:"stepsGroups" validate:"dive"`
 }
```

Comment:
@webrouse rychlej dotaz. 🙂 Kam bys ten seznam komponent ukládal? Začal jsem to přidávat od toho dialogu na vytváření šablony a tak nějak mimochodem mi to spadlo do `inputs.jsonnet`. 😅 Jakoby obdobně se `stepsGroups`. Nicméně ale jak tak koukám na tu strukturu šablony, tak `manifest.jsonnet` asi dává větší smysl, že?